Fix the following error in tests:

    .gem/gems/pry-0.13.1/lib/pry/code.rb:342:in `<class:Code>': undefined method `=~' for class `Pry::Code' (NameError)

--- a/mathematical.gemspec
+++ b/mathematical.gemspec
@@ -30 +30 @@
-  spec.add_development_dependency 'pry-byebug', '~> 3.9.0'
+  spec.add_development_dependency 'pry-byebug', '~> 3.10.1'
